src/olympia/devhub/tasks.py

```diff
@@ -118,6 +118,7 @@ def create_version_for_upload(addon, upload, channel):
             'Skipping Version creation for {upload_uuid} that would '
             ' cause duplicate version'.format(upload_uuid=upload.uuid)
         )
+        return None
```

Comment:
obsolete line?

Reply:
It's just to be explicit, with the `else` clause returning `version` now.

Reply:
I'm surprised there isn't a lint rule about having `return` rather than `return None`